app/components/works/file_row_component.html.erb

```diff
@@ -1,12 +1,12 @@
-<div class="dz-preview dz-file-preview row <%= 'dz-complete' if uploaded? %>" data-dropzone-target="preview">
-  <div class="thumb" class="col-md-1">
+<div class="dz-preview dz-file-preview row pb-3 pt-3 <%= 'dz-complete' if uploaded? %>" data-dropzone-target="preview">
+  <div class="thumb col-1">
     <img data-dz-thumbnail />
   </div>
-  <div class="col-md-2">
-    <% if filename %>
-      <div class="dz-filename" data-dropzone-target="fileName">
-        <span class="fa-solid fa-check file-uploaded" aria-hidden="true"></span>
-        <span><%= filename %></span>
+  <div class="col-5 pt-2">
```

Comment:
Is the pt necessary on a column, since you have it on the row?

Reply:
nope, probably not ... removed